content/guide/07-element-directives.md

```diff
@@ -387,4 +387,173 @@ export default {
 {
 	test: ""
 }
-```
\ No newline at end of file
+```
+
+### Actions
+
+Components can get you far, but there are occasions when you need functionality for an element and wrapping it in a
+component prevents other functionality being added. How do you add a tooltip to the `<Link>` component from your
+favorite routing library?
+
+Actions let you decorate elements with additional functionality.
+
+```html
+<!-- { title: 'Actions' } -->
+Copy link: <input type=text value={url} use:autoselect>
+
+<script>
+	export default {
+		actions: {
+      autoselect(node) {
```

Comment:
it looks like we're mixing and matching tabs and spaces — can we make these all tabs?